### DISCUSSION TITLE
Monsters spawned in swarms have same loot

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13761] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I killed a swarm of ~10 husked crawlers and they all got husk stingers, and i think probability to have stinger is 1/11, very suspicious
And then i checked different monsters, and yes all monsters in some swarm always share same loot

If you spawn them with spawncharacter inventory is randomised as intended

### Reproduction steps

quickstart at 100 difficulty to make swarms bigger
find some monsters, kill them and check their loot

I think you have to encounter them naturally
I tried to spawn them with triggerevent but they seem to spawn with empty pockets

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created a ticket about this in the issue tracker: https://github.com/FakeFishGames/Barotrauma/issues/13768
